### PR TITLE
CFD-494 In the picture : banner on overview pages does not allow flexibility

### PR DESCRIPTION
**Pull request by HelenaEksler**
https://issuetracker.amplexor.com/jira/browse/CFD-494

### CODE REVIEW

**Comment by HelenaEksler** [comment]
Screenshots:
inthepic
@nader77 review, please.

**Comment by zero2one** [comment]
@HelenaEksler needs a re-roll to fix the broken build.

**Comment by HelenaEksler** [comment]
@nader77 Please, review.

File: capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module [hunk 1]
```diff
@@ -299,12 +299,16 @@ function c4m_restful_quick_post_needs_app() {
   // Get the current path.
   $item = menu_get_item();
   $path = $item['path'];
+  foreach(menu_tree('main-menu') as $menu_item) {
```
Comment (nader77) [inline capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module]: ``foreach(`` => ``foreach (``

File: capacity4more/themes/c4m/kapablo/templates/views-view--c4m-overview-inthe-picture-blocks--block.tpl.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            <span class="text">
+              {{ slide.text }}
+            </span>
+          </div>
+        </div>
+      </li>
+    </ul>
+  </div>
+
+
```
Comment (nader77) [inline capacity4more/themes/c4m/kapablo/templates/views-view--c4m-overview-inthe-picture-blocks--block.tpl.php]: Remove line break.

**Comment by nader77** [comment]
@HelenaEksler Looks good other than the minor fixes, Fix and assign to Peter.

**Comment by zero2one** [comment]
@HelenaEksler @nader77 

I see that the link is set only on the header text.

Flexibility is requested: they want to be able to add a in-the-picture block without text (text in the image).
If we create an in-the-picture block without header or description text we and up with a large image but we can not click trough to the link of the in-the-picture item.

Can you make the whole image clickable?

I know that this will remove the swipe functionality from the carousel.
We still have the left & right arrow navigation on the carousel…

**Comment by nader77** [comment]
@zero2one Add link on the whole slide, It won't affect finger swipe in mobile.

**Pull request merged**